src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+
+  --link-hover: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  width: 100vw;
+  justify-content: space-between;
+}
+
+.logo__link {
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 50px;
+  height: 60px;
+  align-items: stretch;
```

Comment:
I believe that stretch value is default one value for `align-items` so you don;t have to explicitly declare it.